monitoring/mongodb/alerts.yaml

```diff
@@ -139,3 +139,45 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbNodeRecovering
+    expr: |
+      count_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"}[1h]) > 0
+    for: 1h
```

Comment:
Do we want to have the same alert after 1 day but with `critical` severity?

Reply:
if it stays for too long (1 day) I think it's appropriate to have it as a critical alert , I believe this is handled by the alert manager

Reply:
Yes, for that you just need to recreate another alert with the same name but another severity, alertmanager will only display the one with the higher severity, so if we have the critical, it won't duplicate what is showed to the user